browser/utils/mdc/bindServiceToApp.js

```
@@ -25,13 +35,37 @@ module.exports = async (appName, serviceName, params) => {
     });
 
     switch (serviceName) {
-        case 'Data Sync':
+        case syncServiceName:
+            expect(params).to.have.key('syncUrl')
             await page.waitForSelector('#root_syncServerConfig_url')
             // Type in Data Sync URL
-            await page.type('#root_syncServerConfig_url', params.syncUrl)
-            break;
+            await page.type('#root_syncServerConfig_url', `https://${params.syncUrl}`)
+            break
+        case idmServiceName:
+            expect(params).to.include.all.keys('idmRealmName', 'idmUser', 'idmPassword')
+            const realmInputId = '#root_realmSettings_realmId'
+            await page.waitForSelector(realmInputId)
+            // Trigger mouse triple-click to select existing text in input field to delete it :)
+            await page.click(realmInputId, { clickCount: 3 })
+            await page.type(realmInputId, params.idmRealmName)
+            await page.type('#root_realmSettings_adminUsername', params.idmUser)
+            await page.type('#root_realmSettings_adminPassword', params.idmPassword)
+            break
+        case mssServiceName:
+            expect(params).to.have.key('mssAppId')
+            await page.waitForSelector('#root_appConfig_appId')
+            await page.type('#root_appConfig_appId', params.mssAppId)
+            break
+        case upsServiceName:
+            expect(params).to.include.all.keys('upsAndroidServerKey', 'upsAndroidSenderId')
+            await page.waitForSelector('#root_CLIENT_TYPE')
+            await page.select('#root_CLIENT_TYPE', 'Android')
+            await page.type('#root_platformConfig_googlekey', params.upsAndroidServerKey)
+            await page.type('#root_platformConfig_projectNumber', params.upsAndroidSenderId)
+            break
         default:
-            break;
+            console.error('Unknown service name')
+            process.exit(1)
```

It's better to throw an error here
```js
throw new Error('Unknown service name')
```